include/script.h

```diff
@@ -56,5 +56,6 @@ void ClearRamScript(void);
 bool8 InitRamScript(u8 *script, u16 scriptSize, u8 mapGroup, u8 mapNum, u8 objectId);
 u8 *GetRamScript(u8 objectId, u8 *script);
 u16 gScriptResult;
+u16 gScriptItemId;
```

Comment:
I doubt this variable is part of the `script` module, based on it's position. Might've been part of `item.o` though. Either way, should be marked `extern`, and other `extern` declarations of it should be removed.